File: formulario_opret_part2/lib/models/respuesta.dart
class Respuesta {
  // int idRespuesta;
  String idUsuarioEmpl;
  String noEncuesta;
  int idPreguntas;
  String? respuestas;
  String? valoracion;

  Respuesta({
    // required this.idRespuesta,
    required this.idUsuarioEmpl,
    required this.noEncuesta,
    required this.idPreguntas,
    this.respuestas,
    this.valoracion
  });

  factory Respuesta.fromJson(Map<String, dynamic> json) {
    return Respuesta(
      // idRespuesta: json['idRespuesta'],
      idUsuarioEmpl: json['idUsuarioEmpl'],
      noEncuesta: json['noEncuesta'],
      idPreguntas: json['idPreguntas'],
      respuestas: json['respuestas'],
      valoracion: json['valoracion']
    );
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};
    // data['idRespuesta'] = idRespuesta;
    data['idUsuarioEmpl'] = idUsuarioEmpl;
    data['noEncuesta'] = noEncuesta;
    data['idPreguntas'] = idPreguntas;
    data['respuestas'] = respuestas;
    data['valoracion'] = valoracion;
    return data;
  }
}

File: WebApi_OPRET/WebApi_OPRET/Repository/Models/Respuesta.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Microsoft.EntityFrameworkCore;

namespace WebApi_OPRET.Repository.Models;

[Table("respuesta")]
public partial class Respuesta
{
    [Key]
    [Column("id_respuesta")]
    public int IdRespuesta { get; set; }

    [Column("id_Usuario_Empl")]
    [StringLength(100)]
    [Unicode(false)]
    public string IdUsuarioEmpl { get; set; } = null!;

    [Column("no_encuesta")]
    [StringLength(100)]
    [Unicode(false)]
    public string NoEncuesta { get; set; } = null!;

    [Column("id_preguntas")]
    public int IdPreguntas { get; set; }

    [Column("respuestas")]
    [StringLength(255)]
    [Unicode(false)]
    public string? Respuestas { get; set; }

    [Column("valoracion")]
    [StringLength(50)]
    [Unicode(false)]
    public string? Valoracion { get; set; }

    [ForeignKey("IdPreguntas")]
    [InverseProperty("Respuestas")]
    public virtual Pregunta IdPreguntasNavigation { get; set; } = null!;

    [ForeignKey("IdUsuarioEmpl")]
    [InverseProperty("Respuestas")]
    public virtual UsuariosEmpl IdUsuarioEmplNavigation { get; set; } = null!;

    [ForeignKey("NoEncuesta")]
    [InverseProperty("Respuestas")]
    public virtual RegistroForm NoEncuestaNavigation { get; set; } = null!;
}
